### ISSUE TITLE
ERROR: useCachedAsyncData is never imported

### ISSUE THREAD

**Issue by waynegibson** [dulnan/nuxt-multi-cache#63]
When trying to import the `useCachedAsyncData`  composable in to nuxt 3 component.

```
<script lang="ts" setup>
import { useCachedAsyncData } from '#imports'

const { data } = await useCachedAsyncData('products:all', () => GqlGetProducts({ first: 2 }),
  {
    serverMaxAge: 5,
    clientMaxAge: 5,

    serverCacheTags: function (data) {
      return data.cacheTags
    },

    transform: function (data) {
      return {
        products: data.products,
        cachedTime: data.currentTime
      }
    }
  }
)

// This is the original request that works without caching
// const { data } = await useAsyncData('products', () => GqlGetProducts({ first: 2 }))
</script>
```

ERROR  Pre-transform error: [unimport] failed to find "useCachedAsyncData" imported from "#imports" 

It was never imported: 

![Screenshot 2024-07-18 at 5 24 40 AM](https://github.com/user-attachments/assets/ebd6e9a6-c09a-4051-8890-e2261ca7a8de)

Version:    "nuxt-multi-cache": "^3.2.0"
Nuxt:  "nuxt": "^3.12.3"
Node:  v20.11.0
PNPM: 9.1.3

**Comment by plcdnl** [comment]
I think it hasn't been released yet. It should be available in future releases

**Comment by bgondy** [comment]
Documentation seems to be a little ahead of time.

![see-you-in-the-future-so-long-gif](https://github.com/user-attachments/assets/2b5637d0-38e5-4dff-bc4e-fcd70407b2f1)

**Comment by plcdnl** [comment]
I think he wanted to release it earlier but maybe he was a little busy. He's doing an incredible job!

**Comment by waynegibson** [comment]
Thank you.  I have left the thread open should anyone else experience the same issue.

**Comment by dulnan** [comment]
I should probably stop auto-deploying the docs from the main branch :sweat_smile: Sorry for the confusion and the long wait! This feature is now available in 3.3.0.
